src/DynamoCoreWpf/Views/PackageManager/Controls/SearchBoxControl.xaml.cs

```diff
@@ -5,21 +5,37 @@
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Data;
+using System.Windows.Threading;
 
 namespace Dynamo.PackageManager.UI
 {
+
     /// <summary>
     /// Interaction logic for SearchBoxControl.xaml
     /// </summary>
     public partial class SearchBoxControl : UserControl
     {
+        private DispatcherTimer delayTimer;
         /// <summary>
         /// Constructor
         /// </summary>
         public SearchBoxControl()
         {
             InitializeComponent();
         }
+        public static DispatcherTimer Debounce(DispatcherTimer dispatcher, TimeSpan interval, Action action)
```

Comment:
`internal` please

Reply:
I guess I will more this method to private, it is good to go ?

Reply:
I moved to private.